modules/communication_module.R

```diff
@@ -215,7 +215,8 @@ communication_server <- function(id,
                    # TODO some check if time is available in data
                    # TODO check if selected sensors has data that time and component, otherwise a message?
                    # TODO for the selected stations and parameters connect with those selection modules
-                   all_data <- isolate(get_data()$data_measurements)
+                   all_data <- isolate(get_data()$data_measurements) %>% 
```

Comment:
This isn't needed if you include all pm-observations in the table, each parameter a column with the number of observations.